views/components/avatar.html

```diff
@@ -0,0 +1,192 @@
+            </li>
+            <li class="drawer-item">
+              <a href="/views/components/padding.html">Padding</a>
+            </li>
+            <li class="drawer-item">
+              <a href="/views/components/margin.html">Margin</a>
+            </li>
+            <li class="drawer-item drawer-item-heading">
+              <a href="/views/components/typography.html"><i class="drawer-item-icon bi bi-file-font"></i>Typography</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <section class="content">
+        <div class="content-body m-2 px-5 box-shadow-medium">
+          <div class="content-body-title border-bottom px-5 py-2 mb-5 border-color">
+            <h2>Avatar</h2>
+          </div>
+          <div class="content-body-details pb-5">
+            <p>
+              <b>JSMonk UI</b> provides different types of
+              <code>avatar</code> components with different sizes. Check the
+              below example.
+            </p>
+            <h3>Avatar circle</h3>
+            <p>
+              You can achieve different size of avatar by just adding
+              <code>av-xs</code>,<code>av-sm</code>,<code>av-md</code>,<code>av-lg</code>
+              and <code>av-xl</code> class as shown below.
+            </p>
+            <div class="demo-component-wrapper m-5">
+              <div class="demo-component-avatar-wrapper">
+                <!-- Avatar -->
+                <img src="/assets/images/avatar.svg" class="avatar av-xs" />
+                <img src="/assets/images/avatar.svg" class="avatar av-sm" />
+                <img src="/assets/images/avatar.svg" class="avatar av-md" />
+                <img src="/assets/images/avatar.svg" class="avatar av-lg" />
+                <img src="/assets/images/avatar.svg" class="avatar av-xl" />
+              </div>
+            </div>
+            <p>
+              <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2509%2509%2509%2509%2509%253C%21--%2520Avatar%2520--%253E%250A%253Cimg%2520src%253D%2522%252Fassets%252Fimages%252Favatar.svg%2522%2520class%253D%2522avatar%2520av-xs%2522%2520%252F%253E%250A%253Cimg%2520src%253D%2522%252Fassets%252Fimages%252Favatar.svg%2522%2520class%253D%2522avatar%2520av-sm%2522%2520%252F%253E%250A%253Cimg%2520src%253D%2522%252Fassets%252Fimages%252Favatar.svg%2522%2520class%253D%2522avatar%2520av-md%2522%2520%252F%253E%250A%253Cimg%2520src%253D%2522%252Fassets%252Fimages%252Favatar.svg%2522%2520class%253D%2522avatar%2520av-lg%2522%2520%252F%253E%250A%253Cimg%2520src%253D%2522%252Fassets%252Fimages%252Favatar.svg%2522%2520class%253D%2522avatar%2520av-xl%2522%2520%252F%253E%250A"
+                style="
+                    width: 656px;
+                    height: 316px;
```

Comment:
a suggestion : make a class if feasible , to give styles to i-frame tag, rest all looks good ! code structure is well